src/state/translations-mixin.ts

```diff
@@ -286,23 +284,6 @@ export default <T extends Constructor<HassBaseEl>>(superClass: T) =>
       configFlow?: Parameters<typeof getHassTranslations>[4],
       force = false
     ): Promise<LocalizeFunc> {
-      if (
-        __BACKWARDS_COMPAT__ &&
-        !atLeastVersion(this.hass!.connection.haVersion, 0, 109)
-      ) {
-        if (category !== "state") {
-          return this.hass!.localize;
-        }
-        const resources = await getHassTranslationsPre109(this.hass!, language);
```

Comment:
I don't think this should have been removed, it is for when the frontend is not bundled with the backend (cast, supervisor) and is only called when the backed version is old.

Reply:
https://github.com/home-assistant/frontend/pull/20597